README.md

```diff
@@ -76,6 +78,7 @@ complete beginners.
 - [learnpython](https://www.learnpython.org/) - Website Which is generously supported by DataCamp where you can learn python.
 - [python.org](https://wiki.python.org/moin/BeginnersGuide/Programmers) - Website that contain various resources for learning python.
 - [Automate the Boring Stuff with Python](https://automatetheboringstuff.com/) - This is one of the websites where you can learn python.
+- [W3Schools.com/Python](https://www.w3schools.com/python/default.asp) - We create simplified and interactive learning experiences.
```

Comment:
@carlosvinimsouza Please can you remove the python link you added from here, and leave it in the python separate file. This is because we only added 5 to our preview and you can see the rest of the items in the separate md file. Thank you.

Reply:
@CodingSpecies Good catch